README.md

```diff
@@ -630,6 +630,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 ## Security and PKI
 
   * [alienvault.com](https://www.alienvault.com/open-threat-exchange/reputation-monitor) — Uncovers compromised systems in your network
+  * [Atom](https://atom.io/) - Atom
```

Comment:
So sorry, missed you added this one - but again its just one word.  It's also just word and its EOL so should be removed.